packages/toolpad-app/src/server/localMode.ts

```diff
@@ -992,7 +992,7 @@ class ToolpadProject {
   constructor(root: string, options: Partial<ToolpadProjectOptions>) {
     this.root = root;
     this.options = {
-      cmd: 'start',
+      // cmd: 'start',
```

Comment:
Is there a reason to leave this in commented, or a miss?

Reply:
a miss